README.md

```diff
@@ -8,305 +8,409 @@
+    * [Service properties](#service-properties)
+    * [Binding properties](#binding-properties)
+    * [Account context in operator secret and configmap](#account-context-in-operator-secret-and-configmap)
+    * [Versions](#versions)
+*   [Contributing](#contributing)
+*   [Troubleshooting](#troubleshooting)
 
-## Requirements
+<!-- SHOW operator hub -->
 
-The operator can be installed on any Kubernetes cluster with version >= 1.11. 
-You need an [IBM Cloud account](https://cloud.ibm.com/registration) and the 
-[IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started).
-You need also to have the [kubectl CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/)  
-already configured to access your cluster. Before installing the operator, you need to login to 
-your IBM cloud account with the IBM Cloud CLI:
+## Features
 
-```bash
-ibmcloud login
-```
+* **Service provisioning**: Create any service with any plan that is available in the [IBM Cloud catalog](https://cloud.ibm.com/catalog#services).
 
-and set a default target environment for your resources with the command:
+* **Bindings management**: Automatically create Kubernetes secrets in your Kubernetes cluster with the credentials to bind to any provisioned service to the cluster.
 
-```bash
-ibmcloud target --cf -g default
-```
+<!-- END SHOW operator hub -->
+[Back to top](#ibm-cloud-operator)
+<!-- SHOW operator hub -->
 
-This will use the IBM Cloud ResourceGroup `default`. To specify a different ResourceGroup, use the following command:
-```bash
-ibmcloud target -g <resource-group>
-```
+## Prerequisites
 
-Notice that the `org` and `space` must be included, even if no Cloud Foundry services will be instantiated.
+1.  Have an [IBM Cloud account](https://cloud.ibm.com/registration).
+2.  Have a cluster that runs Kubernetes version 1.11 or later (OpenShift 3.11 or later).
+3.  Install the required command line tools.
+    *   [IBM Cloud CLI](https://cloud.ibm.com/docs/cli?topic=cloud-cli-getting-started) (`ibmcloud`)
+    *   [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/) (`kubectl`)
+4.  Log in to your IBM Cloud account from the CLI.
+    
+    ```bash
+    ibmcloud login
+    ```
 
-## Set up the operator
+5.  Target the appropriate environment (`--cf`) and resource group (`-g`). Note that you must still set the Cloud Foundry `org` and `space` environment, even if you do not create any Cloud Foundry services.
+    
+    ```bash
+    ibmcloud target --cf -g <default>
+    ```
 
-To set up the operator after logging in with `ibmcloud`, run the below installer.
+6.  Set the Kubernetes context of your CLI to your cluster so that you can run `kubectl` commands. For example, if your cluster runs OpenShift, use the `oc login` command. 
 
-By default, the script will create a new API key for use in the operator. To use a custom API key, set the `IBMCLOUD_API_KEY` environment variable to the key.
```

Comment:
I think we should keep this bit, indicating you can use a custom API key even if it's not a service ID